app/controllers/home_controller.rb

```diff
@@ -1,3 +1,11 @@
 class HomeController < ApplicationController
+  layout "landing"
+
   def index; end
+
+  def about_us; end
+
+  def pricing; end
+
+  def contacts; end
```

Comment:
Do we still need this empty actions? AFAIK there is no need in them if there are templates with appropriate names.

Reply:
very simple templates must be added
with h1 Pricing for example

Reply:
I understand. I'm saying that we don't need those definitions if we have templates. As i remember.

Reply:
Yes works without actions in the controller